build.gradle

```diff
@@ -7,7 +7,7 @@
 apply plugin: 'java'
 
 group = 'com.ca.apim.gateway.assertion'
-version = '1.0.0'
+version = 'sdk811-2.0-2560'
```

Comment:
how did you come up with "2.0-2560"

Reply:
Followed the same pattern in previous assertion: InjectionFilterAssertion-sdk82-1.0-2559.sjar

Bumped major version because of JDK11 update and increased build number

Reply:
The `InjectionFilterAssertion-sdk82-1.0-2559.sjar` naming is likely from the time **before** open sourcing this repo. Let's continue following semantic versioning.

Reply:
> The `InjectionFilterAssertion-sdk82-1.0-2559.sjar` naming is likely from the time **before** open sourcing this repo. Let's continue following semantic versioning.

are you suggesting sdk811-1.0-2560.sjar?

Reply:
No, I mean that `version` variable should follow semver convention, e.g. be '2.0.0' or  '1.1.0'.

Reply:
I think he just means continue from 1.0.0 so either 1.1.0 or 1.0.1, and disregard the versioning before open sourcing.

Reply:
Version set to 1.1.0